controls/cf_serverd.cf

```diff
@@ -32,9 +32,6 @@ body server control
       # Suggested value >= total number of hosts
       maxconnections        => "$(def.control_server_maxconnections)";
 
-      # Bind to all interfaces including ipv6
```

Comment:
If you remove this definition from the implicit any:: context then the agent will only bind to ipv4 interfaces, so this one should remain, and then be overridden for windows below.